rAmAnuja-sampradAyaH/kriyA/venkaTa-nAtha-shAkhA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH/04_ijyA.md

```diff
@@ -2596,158 +2612,306 @@ title = "०४ इज्या-प्रयोगः"
+> माषादि-चूर्णैर् मृद्भिर् वा प्रक्षाल्य करयोर् द्वयम्।  प्रक्षाल्य जानु-पादौ च काष्ठैर् दन्तान् विशोधयेत्॥ 
 
-f 
+रत्नाकरे -- 
+
+> मूत्रे पुरीषे भुक्त्य्-अन्ते भक्ष्याणां भक्षणे ऽपि च।  
+चतुर्-अष्ट द्वि-षट् द्व्य्-अष्ट-गण्डूषैस् तु विशुद्ध्यति॥ [[??]]
+
+> आचान्तो ऽप्य् अशुचिस् तावद् यावत् पात्रम् अनुद्धृतम्।  
+उद्धृते ऽप्य् अशुचिस् तावद् यावन् मण्डल-शोधनम्॥ [[??]]
 
-शाण्डिल्यः- माषादिचूणे र्मृद्भिर्वा वळ्य कर योद्वकायम् । प्रळ्य जानु पादौच क्याष् दशान्ता न्विशोध 
+रत्नाकरे -- 
 
-रत्नाक रे मूले पुरीषे भुक्त्यन्तो भक्ष्याणां भक्ष णेपिच 1 चतुरष्ट द्विषट् द्व्यष्ट गण्डूषेस्तु विशुद्ध्यति 
+> भुक्तोच्छिष्टस् त्व् अनाचान्तश् शूद्रादीन् यदि पश्यति।  प्रायश्चित्तम् अकुर्वाणः तैस् सार्धं भुक्तवान् असौ॥ [[??]]
 
-आचास्तू प्यशुचिस्ता विद्यावत्पात मनुद्धृतम् । उद्बते प्यशुचि साव द्याव न्मण्डल शोधनम् ॥ 
+और्वः -- 
 
-रत्नाक रे- भुच्छिष्ट स्वनाचान्त श्शूदादी न्यदि पश्यति । प्रायश्चित्त मकुर्वाणः तैस्सार्धं भुक्तवा नसौ । 
+> भुक्त्वा सम्यग् अथाचम्य प्राङ्-मुखोदङ्-मुखो[[??]] ऽपि वा।  
+स्वस्थः प्रशान्त-चित्तस् तु कृतासन परिग्रहः॥  
+अभीष्ट-देवतानान् तु कुर्यात् संस्मरणं बुधः॥ 
 
-और्व-- भुक्त्वा सम्य गथाचम्य प्राङ्मभो दङ्मुखोपि ना । स्वस्थः प्रशा स्त्रचित्तस्तुकृतासन परिग्रहः तानान्तु कुर्या त्संस्मरणं बुधः ॥ 
+शातातपः -- 
 
-अभीष्ट देव 
+> आस्यं प्रक्षाल्य गण्डूषं पिबेद् भुक्त्वा च यो द्विजः।  
+दैवं श्राद्धम् असौ हस्यात् पितॄन् आत्मानम् एव च॥
 
-शातातपः- आस्यं प्र्राळ्य गण्डूषं फिबे द्भुक्त्वाच 
+गण्डूष-काले आस्य-स्थ[[??]] जलं न पिबेद् इत्य् अर्थः। 
 
-I 
+कुत्सः -- 
 
-नात्मान 
+> आचान्तो नाभि-देशन् तु प्राणानां ग्रन्थिर् असीति। विमृश्यादित्यम् ईक्षेत रात्रौ दीप-मुखं ततः॥ 
 
-यो द्विजः ! दैवं श्राद्ध मसौ हस्यो तिृत्य ना "मेम च गण्णूषकाले आस्यस्थ जलं न पिबेदित्यर्थः। 
+व्यासः -- 
 
-कुत्सः- आचान्तो नाभि देशन्तु प्र्राणानां गद्धि रसीति ! निमृश्यादित्य विजेत रात्रा दीपमुखं ततः ॥ व्यासः आच म्याङ्गुष्ठमा श्रेति पादाङ्गुष्टेतु दक्षिणे ! निसावये द्धस्तजल मूर्ध्वहस्तस्समाहितः हुतानु मन्त्रणं कुर्या च धायामिति म स्त्रतः । आधाक्षरेण स्वात्मानं योजये दृ ह्मणीतिहि । सर्वेषामेव योगाना मात्मयोगः परस्स्मृ ई । यो नेन विधिना कुर्या त्सयाळी पद मऎ.यम् ॥ 
+> आचम्याङ्गुष्ठ-मात्रेति पादाङ्गुष्ठे तु दक्षिणे।  
+निस्रावयेद् धस्त-जलम् ऊर्ध्व-हस्तस् समाहितः॥  
+हुतानुमन्त्रणं कुर्याच् छ्रद्धायामिति मन्त्रतः।  
+अथाक्षरेण स्वात्मानं योजयेद् ब्रह्मणीति हि॥  
+सर्वेषाम् एव योगानाम् आत्म-योगः परस् स्मृतः।  
+यो ऽनेन विधिना कुर्यात् स याति पदम् अक्षयम्॥ 
 
 [[260]]
 
-1 
+बाहटः -- 
 
-बाहट-- भुक्त्वोप विशत स्तुन्द मोजो भवति तिष्ठतः । आयुश्चन्ध्रमतो मृत्यु र्धावत स्स्वपतोवपुः ! श्रीविष्णुपुरा णे- अग्नि राप्यायये द्धातुं पार्थिवम्पवनेरितः ! दत्ता वकाशं सभना जरयत्वस्तु मोसुखम् अन्नं बणाय मे भूमे रपामग्न्य निलस्यच । भवत्वेतत्परिणतं ममास्त्व विरतं सुखम् 
+> भुक्त्वोपविशतस् तुन्दम् ओजो भवति तिष्ठतः।  
+आयुश्-चङ्क्रमतो[[??]] मृत्युर् धावतस्स स्वपतो वपुः॥ 
+
+श्री-विष्णु-पुराणे -- 
+
+> अग्निर् आप्याययेद् धातुं पार्थिवं पवनेरितः।  
+दत्तावकाशं सभना[[??]] जरयत्व् अस्तु[[??]] मे सुखम्॥  
```

Comment:
https://srivaishnavan.com/publications/meghamala/puranas/vishnu-purana-content/amsa-03/%E0%A4%B6%E0%A5%8D%E0%A4%B0%E0%A5%80%E0%A4%B5%E0%A4%BF%E0%A4%B7%E0%A5%8D%E0%A4%A3%E0%A5%81%E0%A4%AA%E0%A5%81%E0%A4%B0%E0%A4%BE%E0%A4%A3%E0%A4%AE%E0%A5%8D-amsa-03-ady-11/ अत्रान्विष्य पूरयतु